### ISSUE TITLE
Getting error during scanning

### ISSUE THREAD

**Issue by prashantvidja** [segofensiva/OSfooler-ng#2]
Hello,

When I have start scanning using below command I got below error on console.
`osfooler-ng --os-nmap="D-Link DES-3200 switch" -i enp0s8`

![Screenshot from 2019-07-17 15-37-19](https://user-images.githubusercontent.com/26060662/61367232-d3a09780-a8a8-11e9-8bae-f3c8388fc8dd.png)

Is there anything i am missing?

Thanks
prashant

**Comment by moonbaseDelta** [comment]
Hi there, I guiess I found the solution for that case and some others - it's about version of nfqueue used by the lib. I rework some lines, and now it seems to be working fine on other distros as well. Gonna make it as fork, so check if my solution helps

**Comment by prashantvidja** [comment]
Hi @moonbaseDelta,

Thank you very much for this help,
But now when I am scanning using Nmap every time I am not getting any os name I am getting only 
Below results. I have tried multiple fingerprints but getting the same for all.
It does not show any selected os in the result.

Getting below string in scan output.
**"Too many fingerprints match this host to give specific OS details"**

![Screenshot from 2019-07-31 12-00-49](https://user-images.githubusercontent.com/26060662/62188779-0c516e00-b38b-11e9-87af-bada1bd4046d.png)

Thanks.

**Comment by moonbaseDelta** [comment]
Hm, does p0f show same results?

**Comment by prashantvidja** [comment]
Yes, I have just tried using p0f it detects that os is Linux but in osfooler, I have set Windows.

**Comment by moonbaseDelta** [comment]
Well, it could be related to VM or even home router, try check youself without OSfooler for that case. You may go with another VM with special OS: if p0f matches it exatly as your main VM then the issue on the network.
